lib/mintel/rules/haproxy-ingress.libsonnet

```diff
@@ -0,0 +1,49 @@
+{
+  prometheusRules+:: {
+    groups+: [
+      {
+        name: 'haproxy-ingress.rules',
+        rules: [
+          {
+            expr: 'label_replace(\n  label_replace(http_backend_response_wait_seconds_bucket{backend!~"(error|stats|httpback|upstream).*"}, "mintel_com_service", "$1", "backend", "(.*)-\\\\d+$")\n    * on(mintel_com_service) group_left(label_app_kubernetes_io_owner)\n    label_join(kube_service_labels, "mintel_com_service", "-", "namespace", "service"),\n  "label_app_kubernetes_io_owner",\n  "satoshi",\n  "label_app_kubernetes_io_owner",\n  ""\n)\n',
```

Comment:
i think those rules with escaped "\n" and similar are bad and will be hard to edit going forward.

I don't think they need to be, expr can be multiline right ?

Reply:
yeah, let me fix the haproxy ones

Reply:
Should be good now. Also replaced "\\\\d" with "\\d"